lms/static/sass/base/_base.scss

```diff
@@ -130,7 +130,6 @@ a:focus {
   margin: 0 auto 0;
   padding: ($baseline*2) 0;
   max-width: grid-width(12);
-  min-width: 760px;
```

Comment:
@marjev, hmmm. Looking at https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Fshared%2F_course_object.scss, the styling here looks to be related to the very old homepage/course discovery views. In the short term, I think its fine to move this specific container-based override into that file, but you'll still need to scope it to a specific view (either by a specific class only used by this view - e.g. ``container-course`` or by assigning a view/layout-based class to the ``<body>`` element).